extensions/skins/conversation_v1/Conversation.js

```diff
@@ -208,11 +208,22 @@ oppia.directive('conversationSkin', [function() {
       $scope.PANEL_TUTOR = 'tutor';
       $scope.PANEL_SUPPLEMENTAL = 'supplemental';
 
+      $scope.helpHtml = null;
+      $scope.continueToNextCardHtml = null;
+
       $scope.profilePicture = '/images/avatar/user_blue_72px.png';
       oppiaPlayerService.getUserProfileImage().then(function(result) {
         $scope.profilePicture = result;
       });
 
+      $scope.clearHelpHtml = function() {
+        $scope.helpHtml = null;
+      };
+
+      $scope.clearContinueToNextCardHtml = function() {
+        $scope.continueToNextCardHtml = null;
+      };
+
```

Comment:
If we had just one variable that contained the content, we could have just one clearing function.